katsdpdatawriter/spead_write.py

```diff
@@ -81,51 +81,56 @@ def _dtype_converter(dtype: Any) -> np.dtype:
 
 def io_sensors() -> Sequence[Sensor]:
     """Create input and output counter sensors."""
+    def make_sensor(*args, **kwargs) -> Sensor:
+        kwargs['auto_strategy'] = SensorSampler.Strategy.EVENT_RATE
+        kwargs['auto_strategy_parameters'] = (0.05, 10.0)
```

Comment:
Do we need as much as 20Hz lower bound for these sensors, I would imagine 5Hz would be plenty (in line with keeping load down).

Reply:
I'd like to start with a slightly higher rate and then throttle down as needed. If we start with a low sampling rate and discover during a post-mortem that we should have had it higher then it will be too late.

Reply:
Fair enough